.github/CODEOWNERS

```diff
@@ -0,0 +1,354 @@
+/flang/test/Driver @llvm/pr-subscribers-flang-driver
+
+# m68k
+/llvm/lib/Target/M68k @llvm/pr-subscribers-m68k
+/clang/lib/Basic/Targets/M68k.* @llvm/pr-subscribers-m68k
+/clang/lib/CodeGen/Targets/M68k.cpp @llvm/pr-subscribers-m68k
+
+# libcxx
+/libcxx/ @llvm/pr-subscribers-libcxx
+/runtimes/ @llvm/pr-subscribers-libcxx
+
+# libcxxabi
+/libcxxabi/ @llvm/pr-subscribers-libcxxabi
+/runtimes/ @llvm/pr-subscribers-libcxxabi
+
+# libunwind
+/libunwind/ @llvm/pr-subscribers-libunwind
+/runtimes/ @llvm/pr-subscribers-libunwind
+
+# objectyaml
+/llvm/include/llvm/ObjectYAML/ @llvm/pr-subscribers-objectyaml
+/llvm/lib/ObjectYAML/ @llvm/pr-subscribers-objectyaml
+/llvm/test/tools/obj2yaml/ @llvm/pr-subscribers-objectyaml
+/llvm/test/tools/yaml2obj/ @llvm/pr-subscribers-objectyaml
+/llvm/tools/obj2yaml/ @llvm/pr-subscribers-objectyaml
+/llvm/tools/yaml2obj/ @llvm/pr-subscribers-objectyaml
+
+# clang-analysis
+/clang/include/clang/Analysis/ @llvm/pr-subscribers-clang-analysis
+/clang/lib/Analysis/ @llvm/pr-subscribers-clang-analysis
+
+# clang-static-analyzer
+/clang/include/clang/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/lib/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/tools/scan-build/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/utils/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/docs/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+
+# pgo
+/llvm/lib/Transforms/Instrumentation/ @llvm/pr-subscribers-pgo
+/llvm/test/Instrumentation/ @llvm/pr-subscribers-pgo
+/compiler-rt/lib/profile/ @llvm/pr-subscribers-pgo
+/compiler-rt/lib/memprof/ @llvm/pr-subscribers-pgo
+/compiler-rt/test/profile/ @llvm/pr-subscribers-pgo
+/compiler-rt/test/memprof/ @llvm/pr-subscribers-pgo
+/llvm/tools/llvm-profdata/ @llvm/pr-subscribers-pgo
+/llvm/tools/llvm-profgen/ @llvm/pr-subscribers-pgo
+/llvm/test/tools/llvm-profdata/ @llvm/pr-subscribers-pgo
+/llvm/test/tools/llvm-profgen/ @llvm/pr-subscribers-pgo
+/llvm/unittests/ProfileData/* @llvm/pr-subscribers-pgo
+
+# openacc
+/flang/**/OpenACC/ @llvm/pr-subscribers-openacc
+/flang/include/flang/Lower/OpenACC.h @llvm/pr-subscribers-openacc
+/flang/docs/OpenACC.md @llvm/pr-subscribers-openacc
+/flang/lib/Parser/openacc-parsers.cpp @llvm/pr-subscribers-openacc
+/flang/lib/Lower/OpenACC.cpp @llvm/pr-subscribers-openacc
+/llvm/**/Frontend/OpenACC/ @llvm/pr-subscribers-openacc
+/llvm/unittests/Frontend/OpenACCTest.cpp @llvm/pr-subscribers-openacc
+/mlir/test/Target/LLVMIR/openacc-llvm.mlir @llvm/pr-subscribers-openacc
+/mlir/test/Dialect/OpenACC @llvm/pr-subscribers-openacc
+/mlir/**/Dialect/OpenACC/ @llvm/pr-subscribers-openacc
+/mlir/**/Conversion/OpenACCToSCF/ @llvm/pr-subscribers-openacc
+/mlir/lib/Target/LLVMIR/Dialect/OpenACC/ @llvm/pr-subscribers-openacc
+/mlir/lib/Conversion/OpenACCToSCF @llvm/pr-subscribers-openacc
+/mlir/lib/Conversion/OpenACCToSCF/OpenACCToSCF.cpp @llvm/pr-subscribers-openacc
+/mlir/lib/Dialect/OpenACC/ @llvm/pr-subscribers-openacc
```

Comment:
@clementval : can't all of these be folded in `**/*OpenACC` ? 
(or `mlir/**/*OpenACC` if you want to be more targeted per folder)

Reply:
I guess it can be but shouldn't it be `mlir/**/*OpenACC*`?

Reply:
I assumed we match on prefix?  Otherwise all the other patterns are incorrect as well. 

Checking with `.gitignore`:
```
$ mkdir -p mlir/foo/barOpenACCbaz/blah
$ git status
...
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	mlir/lib/Dialect/OpenACC/empty
$ echo 'mlir/**/*OpenACC' >> .gitignore 
$ git status
```

It seems to work as is.

Reply:
Yeah then it should be good. I added a suggested change.